### ISSUE TITLE
Wrong initial position of popper element when reference element is a child of a sticky element.

### ISSUE THREAD

**Issue by soapsign** [FezVrasta/popper.js#253]
### CodePen demo

https://codepen.io/soapsign/pen/qmpWOG

### Steps to reproduce the problem

1.  Scroll the `container` div
2.  Click anywhere
3.  The initial position of the popper element is wrong

### What is the expected behaviour?

The position of the popper element should be at the bottom of the reference element.

### What went wrong?

The reference element is a child element of a `position: sticky;` element. The initial position of the popper element does not take this into account. After the popper element is displayed, scrolling the
`container` div afterward correct the the wrong position.

### Any other comments?

Any additional information.

**Comment by FezVrasta** [comment]
I investigated a bit, the problem seems to rely in `preventOverflow`, specifically in the boundaries computation.

**Comment by soapsign** [comment]
Have make some investigation about it lately. It turns out that `getBoundingClientRect()` on `Element` will return incorrect result when HTML changes.  

Firefox have a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=672457) reported about it and seems has been fixed. 

I have tested on `Safari(Version 10.0)` and `Firefox(53.0.2)` and it seem to be working fine. The confirmed browser that have this problem is `Chrome(Version 59.0.3071.109)`.
